pyomo/solvers/tests/mip/test_solver.py

```diff
@@ -32,7 +32,7 @@ def tearDownModule():
     TempfileManager.tempdir = old_tempdir
 
 
-class TestSolver2(pyomo.opt.OptSolver):
+class DummySolver2(pyomo.opt.OptSolver):
```

Comment:
Just a suggestion: I think it's more common to refer to these as "mock" objects in Python. I'm fine with `DummySolver` but we may want to consider `MockSolver`